### ISSUE TITLE
What version of these npm packages should be used together?  (igniteui-angular/@angular/core/@angular/cli)

### ISSUE THREAD

**Issue by brookhutchinson** [IgniteUI/igniteui-angular#7584]
## Question  
What version of these npm packages should be used together?
@angular/cli
@angular/core
igniteui-angular

Examples
If I would like to use **igniteui-angular@8.2.30** then
What version of @angular/cli should be used?
What version of @angular/core should be used?

If I would like to use **igniteui-angular@9.0.0** then
What version of @angular/cli should be used?
What version of @angular/core should be used?

If I would like to use **igniteui-angular@9.0.17** then
What version of @angular/cli should be used?
What version of @angular/core should be used?

If I would like to use **igniteui-angular@9.1.0** then
What version of @angular/cli should be used?
What version of @angular/core should be used?

 * igniteui-angular version: 9.0.17
 * browser: Version 83.0.4103.97 (Official Build) (64-bit)

**Comment by kdinev** [comment]
@brookhutchinson The major version number of `igniteui-angular` corresponds to the major version of `@angular/core` that should be used with it. E.g. `igniteui-angular@8.x` works with any `@angular/core@8.x`

@damyanpetev will respond with the specific `igniteui-cli`.

**Comment by Lipata** [comment]
Hi, @brookhutchinson.

igniteui-cli@4.x works with @angular/core@8.x
igniteui-cli@5.x works with @angular/core@9.x

**Comment by damyanpetev** [comment]
I guess I'll expand on both answers then 🙂
Since version 6 the Angular team unified most packages to a single **major** version. So `@angular/core@9.x.x` (and so on) go with `@angular/cli@9.x.x`. They might differ slightly on minor/patch version, but are intended to be compatible. So I'll refer to these packages as a single Angular version from now on.

The [versioning of Ignite UI for Angular](https://github.com/IgniteUI/igniteui-angular/wiki/Ignite-UI-for-Angular-versioning) also follows a similar approach where our major version is tied to the Angular version the library was compiled with and supports.
- So if you have `igniteui-angular@8.2.30` it should play nice with Angular 8 packages.
- Same with `igniteui-angular@9.0.0` - it now targets Angular 9 packages.
- The patch/minor increments like `9.0.17` or `9.1.0` still target Angular 9 regardless. You can see that in our peer dependencies (as in "works with") that still require [`"@angular/core": "^9.0.0"`](https://github.com/IgniteUI/igniteui-angular/blob/501e1753819d1d30eef720cefb5c2a79811812a3/projects/igniteui-angular/package.json#L78) which matches all Angular 9 version.

As for our Ignite UI CLI the versioning is separate as we accommodate for other products and may need to version up for those. We have, however, introduced specialized [Angular Schematics](https://www.infragistics.com/products/ignite-ui-angular/angular/components/general/cli-overview.html) that provide a subset of Ignite UI CLI functionality only for Ignite UI for Angular. Those use similar versioning scheme matching `igniteui-angular@9.1.0` with `@igniteui/angular-schematics@9.1.x`.

Basically, what you should be seeing in your project are all 9-s or all 10-s and so on :)
